### PR TITLE
Feature website heatmap

### PR DESCRIPTION
**Pull request by phalado**

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
#### Pull request under review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi Team!  Great job!  

#### Required changes:
##### Navigation bar

* [x] It should use `div` and positions elements as at the source page (content is not important).

##### Footer

* [x] It should contain centered links (content is not important).

Very good that you use of the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links and the [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer. But also good practise to use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document. 

Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!

### Make these changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding 😄*

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# design-teardown
-Smashing Magazine Design Teardown
+# HTML Forms
+
+Design Teardown is the fourth project in the Microverse curriculum. It consists of building a greyscale heat map of [Smashing Magazine's home page](https://www.smashingmagazine.com/) indicating wich elements have the greatest weight in the visual hierarchy.
```
Comment (Anna-Myzukina) [inline README.md:3]: Please check, I see you use here HTML Forms instead of Design Teardown.

**Review by suretrust — APPROVED** [review]
Hi team, 

### Amazing job guys! Let's get this merged! :+1: 

_Good luck with your future projects and happy coding! :tada:_

**Pull request merged**